src/components/fileSelect/FileSelect.stories.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2021 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { storiesOf } from "@storybook/react";
+import {action} from "@storybook/addon-actions";
+import FileSelect from "./FileSelect";
+
+storiesOf("FileSelect", module)
+    .add("Default", () => (
+        <div>
+            Default File select. Can select one file of any kind.
```

Comment:
```suggestion
            {"Default File select. Can select one file of any kind."}.            
```
Same comment for all strings